.circleci/unittest/windows/scripts/install.sh

```diff
@@ -26,7 +26,8 @@ else
 fi
 
 printf "Installing PyTorch with %s\n" "${cudatoolkit}"
-conda install -y -c "pytorch-${UPLOAD_CHANNEL}" "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}" pytest
+# why add -c conda-forge again, see https://github.com/pytorch/vision/issues/4458
+conda install -y -c "pytorch-${UPLOAD_CHANNEL}" -c conda-forge "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}" pytest
```

Comment:
It's still unclear to me why are you adding conda-forge channel here?

Reply:
@prabhat00155 
without conda-forge, CI would throw exception that torch+cu11.1 isn't in that repo.

Reply:
This link was used. I don't understand why there's a commit to remove all conda-forge. It's a right way to install torch+cu11.1 in CI.

Reply:
https://app.circleci.com/pipelines/github/pytorch/vision/10705/workflows/25d6740c-6c8a-42fd-8cbf-7a54222ebb38/jobs/816118

```
Installing PyTorch with cudatoolkit=11.1
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - cudatoolkit=11.1

Current channels:

  - https://conda.anaconda.org/pytorch-nightly/win-64
  - https://conda.anaconda.org/pytorch-nightly/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch
```

Reply:
Okay, we need `conda-forge` on Windows for cu 11.1. Could you please update the comment you have on line 29 saying that: "`conda-forge` channel is required for cudatoolkit 11.1 on Windows."?

Reply:
Sure, updated!